spectralevents_functions.py

```diff
@@ -21,22 +21,22 @@
 def spectralevents_ts2tfr (S,fVec,Fs,width):
     # spectralevents_ts2tfr(S,fVec,Fs,width);
     #
-    # Calculates the TFR (in spectral power) of a time-series waveform by 
-    # convolving in the time-domain with a Morlet wavelet.                            
```

Comment:
I think my text editor removed excess white space that has no pertinence to the aim of this PR, apologies for the huge diff.